content/sensu-go/5.14/reference/entities.md

```diff
@@ -18,13 +18,13 @@ menu:
   - [Proxy entity labels](#proxy-entities-managed)
   - [Agent entity labels](#agent-entities-managed)
 - [Entities specification](#entities-specification)
-	- [Top-level attributes](#top-level-attributes)
+  - [Top-level attributes](#top-level-attributes)
   - [Spec attributes](#spec-attributes)
   - [Metadata attributes](#metadata-attributes)
-	- [System attributes](#system-attributes)
-	- [Network attributes](#network-attributes)
-	- [NetworkInterface attributes](#networkinterface-attributes)
-	- [Deregistration attributes](#deregistration-attributes)
+  - [System attributes](#system-attributes)
+  - [Network attributes](#network-attributes)
+  - [NetworkInterface attributes](#networkinterface-attributes)
+  - [Deregistration attributes](#deregistration-attributes)
```

Comment:
Ignore this doc -- I can't seem to get the changes reverted locally to drop it from the PR.